src/seedsigner/gui/screens/tools_screens.py

```diff
@@ -171,6 +171,73 @@ def update_title(self) -> bool:
 
 
 
+@dataclass
+class ToolsCoinEntropyEntryScreen(KeyboardScreen):
+    def __post_init__(self):
+        # Override values set by the parent class
+        # TRANSLATOR_NOTE: Inserts the number of flips required
+        self.title = _("Coin Flip 1/{}").format(self.return_after_n_chars)
+
+        # Specify the keys in the keyboard
+        self.rows = 2
+        self.cols = 2
+        self.key_height = GUIConstants.get_top_nav_title_font_size() + GUIConstants.EDGE_PADDING/4 + 2*GUIConstants.EDGE_PADDING
```

Comment:
Always force any division for pixel-related placement to `int`. There's no guarantee that different pixel-level constants will always be even numbers.